app/src/org/commcare/android/fragments/SetupEnterURLFragment.java

```diff
@@ -0,0 +1,85 @@
+package org.commcare.android.fragments;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.support.v4.app.Fragment;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import org.commcare.android.framework.ManagedUi;
+import org.commcare.android.framework.UiElement;
+import org.commcare.dalvik.R;
+import org.javarosa.core.services.locale.Localization;
+
+/**
+ * Created by dancluna on 3/17/15.
```

Comment:
Would be nice to include some documentation on what this class does.
Something simple like "Activity for inputting app installation URL; gives the user the ability to enter a bitly, .., or raw URL.